istio-control/istio-autoinject/values.yaml

```diff
@@ -24,6 +24,10 @@ sidecarInjectorWebhook:
   nodeSelector: {}
   tolerations: []
 
+  # If set, will use the value as injection label. The value must match the 'release' label of the injector,
+  # except when 1.2 istio-injection label is used, which must be set to "enabled".
+  injectLabel: istio-injection
```

Comment:
If we do this don;t we need to replace `  istio-injection: enabled` with `{{injectLabel}}: enabled` on L64 of webhook?